src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/NodeBuilder.cs

```diff
@@ -160,6 +161,10 @@ public void StartAll()
             {
                 node.Start();
             }
+
+            // disabled logs for testing.
+            LogManager.Configuration.LoggingRules.Clear();
```

Comment:
maybe provide a bool in StartAll so it can be manually enabled for tests debuging?

Reply:
isn't DisableLogging enough?
https://nlog-project.org/documentation/v4.5.0/html/M_NLog_LogManager_DisableLogging.htm

so if a test need it, you can call EnableLogging again and DisableLogging at the end

Reply:
Agreed. Very good suggestion. @dangershony what do you think?

I'll block the PR so it's not merged just yet.